src/main/drivers/accgyro/accgyro_spi_bmi160.c

```diff
@@ -139,6 +143,20 @@ static void BMI160_Init(const extDevice_t *dev)
     BMI160InitDone = true;
 }
 
+static uint8_t getBmiOversampleMode()
```

Comment:
```suggestion
static uint8_t getBmiSampleMode()
```

Reply:
In normal it's not oversampling, so this should be called `getBmiSampleMode` imho

Reply:
I would prefer to stick with Oversample mode to match the BMI datasheet. That way if you search for Oversample mode you'll find the right info immediately.

Reply:
`GetBmiOsrMode()` is fine to me.

Reply:
I've changed it to `getBmiOsrMode()`

Reply:
Thanks. Please rebase and squash commits